.github/workflows/spread.yaml

```diff
@@ -38,13 +45,16 @@ jobs:
         with:
           go-version: '>=1.17.0'
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
+      - name: Install LXD
+        uses: canonical/setup-lxd@v0.1.1
+        with:
+            channel: 5.21/stable
```

Comment:
Why not `latest/stable`?

Reply:
because that's a moving target, meaning that this ad-hoc allocation script could start failing suddenly upon a new release